tests/strace--always-show-pid.test

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -efu
+#
+# Check decoding of --argv0 option.
+#
+# Copyright (c) 2023 The strace developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+. "${srcdir=.}/init.sh"
+
+args=
+run_prog ../fchdir > /dev/null
+run_strace --always-show-pid $args > /dev/null
```

Comment:
Since `run_strace` uses `strace -o`, this test would pass without `--always-show-pid`.
In other words, this test doesn't test whether the new option works.

Reply:
In fact, PID prefix is only shown when combining `-f` and `-o`[1]. So this test fails without `--always-show-pid`

[1]: https://github.com/strace/strace/blob/f65d243ba6d0b3885540eca000c00d527c3b028f/src/strace.c#L873